test/unit/services/workers/test-send-accepted-claim-notification.js

```diff
@@ -16,11 +17,23 @@ const sendAcceptedClaimNotification = proxyquire('../../../../app/services/worke
 })
 
 describe('services/send-accepted-claim-notification', function () {
+  var claimId
+
+  beforeEach(function () {
+    return testHelper.insertClaimEligibilityData('IntSchema', reference)
+      .then(function (ids) {
+        // Get claim ID that we can use for the test
+        claimId = ids.claimId
+      }
+    )
+  })
+
   it('should call send-notification with correct details', function () {
```

Comment:
you will need to stub and mock the data being returned by your data call and test it is called